scargo/file_generators/templates/conan/profile_atsam.j2

```diff
@@ -1,22 +1,16 @@
-target_host=arm-none-eabi
-standalone_toolchain=arm-none-eabi
-cc_compiler=gcc
-cxx_compiler=g++
-
 [settings]
 os=baremetal
-arch=armv8
+arch=armv6
 compiler=gcc
 compiler.version=10
 compiler.libcxx=libstdc++
 compiler.cppstd={{ config.project.cxxstandard }}
-arch_build=x86_64
 build_type={{ config.profiles[profile].cmake_build_type }}
 
-[env]
+[buildenv]
 CONAN_CMAKE_FIND_ROOT_PATH=/opt/gcc-arm-none-eabi/lib/
-CC=$target_host-$cc_compiler
-CXX=$target_host-$cxx_compiler
+CC=arm-none-eabi-gcc
```

Comment:
shall we hardcoded it? I thought that it was taken form toml file

Reply:
It's used only for x86 projects as I see it